prep-css-layout-classes/index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CSS Layout Classes</title>
+  <link rel="stylesheet" href="reset.css">
+  <style>
+    /* stylelint-disable block-no-empty */
+    body {
+      font-family: sans-serif;
+    }
+
+    h1 {
+      text-align: center;
+    }
+
+    .container {
+      max-width: 992px;
+      margin: auto;
+     }
+
+    .row {
+      width: 100%;
+      display: flex;
+      text-align: center;
+      justify-content: space-evenly;
+
+    }
+
+    .column {
+      flex-basis: calc(100%/3);
+      padding-left: 8px;
+      padding-right: 8px;
+
+    }
+
+
+  </style>
+</head>
+<body>
+  <h1>Avengers</h1>
+  <div class="container">
+    <div class="row">
+      <div class="column"><img src="https://m.media-amazon.com/images/M/MV5BNDYxNjQyMjAtNTdiOS00NGYwLWFmNTAtNThmYjU5ZGI2YTI1XkEyXkFqcGdeQXVyMTMxODk2OTU@._V1_FMjpg_UX1000_.jpg" alt="Avengers" width="550" height="600"></div>
+      <div class="column"><img src="https://terrigen-cdn-dev.marvel.com/content/qa/1x/5ae4ca-MLou_13x19_DolbyAtAMC_Handout_v4_online_lg.jpg" alt="Infinity war" width="550" height="600"></div>
+      <div class="column"><img src="https://encrypted-tbn2.gstatic.com/images?q=tbn:ANd9GcQA_-tL18_rj9zEcjN6n41NEaJm-kRNF9UeOtvksZ4z_OW6jRA9" alt="Endgame" width="550" height="600" ></div>
+    </div>
+  </div>
+
+  <div class="Container">
+    <div class="row">
+      <div class="column"><h3>Avengers</h3></div>
+      <div class="column"><h3>Avengers-Infinity Game</h3></div>
+      <div class="column"><h3>Avengers-End Game</h3></div>
+    </div>
+  </div>
+
+  <div class="Container">
+    <div class="row">
+      <div class="column">
+        <p>Earth's mightiest heroes must come<br> together and learn to fight as a team<br>if they are going to stop the mischievous<br> Loki
+          and his alien army from enslaving<br> humanity.</p></div>
+      <div class="column">
+        <p>Iron Man, Thor, the Hulk and the rest of the Avengers unite to battle their most <br>powerful enemy yet -- the evil Thanos.
+          On a mission to collect all six Infinity <br>Stones,Thanos plans to use the artifacts to inflict his twisted will on
+          reality.<br> The fate of the planet and existence itself has never been more uncertain<br> as everything the Avengers have
+          fought for has led up to this moment.</p></div>
+      <div class="column"><p>After the devastating events of Avengers: Infinity War (2018), the universe is in <br>ruins. With the help of remaining
+        allies, the Avengers assemble once more in order to<br> reverse Thanos' actions and restore balance to the universe.</p></div>
+    </div>
```

Comment:
Hi Aysha, for this assignment we want to only have one container and one row. The row can have 3 columns and each column will have an image, a heading, and a paragraph.

```html
<div class="container">
  <div class="row">
    <div class="column">
    </div>
    <div class="column">
    </div>
    <div class="column">
    </div>
  </div>
</div>
```